packages/turf-boolean-within/test.js

```diff
@@ -0,0 +1,33 @@
+const glob = require('glob');
+const path = require('path');
+const test = require('tape');
+const load = require('load-json-file');
+const shapely = require('boolean-shapely');
+const booleanJSTS = require('boolean-jsts');
+const within = require('./');
+
+test('turf-boolean-within', t => {
+    // True Fixtures
+    glob.sync(path.join(__dirname, 'test', 'true', '**', '*.geojson')).forEach(filepath => {
+        const {name} = path.parse(filepath);
+        const geojson = load.sync(filepath);
+        const [feature1, feature2] = geojson.features;
+        const result = within(feature1, feature2);
+        if (process.env.JSTS) t.true(booleanJSTS('within', feature1, feature2), '[true] JSTS - ' + name);
+
+        if (process.env.SHAPELY) shapely.within(feature1, feature2).then(result => t.true(result, '[true] shapely - ' + name));
```

Comment:
I'm going to try to make this look like this:

```js
t.true(shapely.within(feature1, feature2), '[true] shapely - ' + name);
```

Reply:
That would be nicer :)

Reply:
Agreed! I got to get on that...